subsys/net/lib/sockets/sockets.c

```diff
@@ -1133,6 +1133,16 @@ int zsock_setsockopt_ctx(struct net_context *ctx, int level, int optname,
 			return 0;
```

Comment:
Please use "Fixes #xxxxx" keyword in the commit message. According to https://help.github.com/en/articles/closing-issues-using-keywords the "Fix" might work here but in all the other places we have "Fixes" string.

Reply:
This is good to know. Thanks!